### PR TITLE
[Profiler] Remove DD_DOTNET_PROFILER_HOME

### PR DESCRIPTION
**Pull request by chrisnas**
Cleanup doc and launchsettings for demo apps

## Summary of changes
Remove DD_DOTNET_PROFILER_HOME and cleanup launchsettings

## Reason for change
- this environment variable is no more used 
- make all demo apps launchsettings consistent with all profilers

## Test coverage
no impact on code

### CODE REVIEW

File: profiler/README.MD [hunk 1]
```diff
@@ -50,21 +46,7 @@ To build C++ projects
 - git
 
 #### Building from the command line
-1. Build C# projects
```
Comment (andrewlock) [inline profiler/README.MD:53]: Just my 2 cents, but I think we should be encouraging the use of Nuke, as that's the only thing that's really tested.

File: profiler/src/Demos/NoOpWcfService/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+      "commandName": "Executable",
+      "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\NoOpWcfService\\$(TargetFramework)\\Datadog.Demos.NoOpWcfService.exe",
+      "nativeDebugging": true
+    },
+    "NoOpWcfService+Profiler": {
       "commandName": "Executable",
       "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\NoOpWcfService\\$(TargetFramework)\\Datadog.Demos.NoOpWcfService.exe",
       "environmentVariables": {
         "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
```
Comment (andrewlock) [inline profiler/src/Demos/NoOpWcfService/Properties/launchSettings.json:13]: Obviously up to you, but IMO you'd be better off using the native loader in these demos etc too

File: profiler/src/Demos/Samples.Computer01/Properties/launchSettings.json [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
         "CORECLR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
         "CORECLR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
-        "CORECLR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
 
         "COMPlus_EnableDiagnostics": "1",
 
         "DD_PROFILING_ENABLED": "1",
-        "DD_DOTNET_PROFILER_HOME": "$(BuildOutputRoot)\\bin\\$(Configuration)-AnyCPU\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Managed\\",
+        "DD_PROFILING_CPU_ENABLED": "1",
+        "DD_PROFILING_EXCEPTION_ENABLED": "1",
```
Comment (andrewlock) [inline profiler/src/Demos/Samples.Computer01/Properties/launchSettings.json:26]: Was this intentionally missing?

```suggestion
        "DD_PROFILING_EXCEPTION_ENABLED": "1",
        "DD_PROFILING_ALLOCATION_ENABLED": "1",
```

File: profiler/src/Demos/Samples.ExceptionGenerator/Properties/launchSettings.json [hunk 4]
```diff
@@ -1,21 +1,32 @@
...
-        "COMPlus_EnableDiagnostics": "1",
         "CORECLR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
+        "CORECLR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
         "CORECLR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
+
+        "COMPlus_EnableDiagnostics": "1",
+
         "DD_PROFILING_ENABLED": "1",
+        "DD_PROFILING_CPU_ENABLED": "1",
+        "DD_PROFILING_EXCEPTION_ENABLED": "1",
```
Comment (andrewlock) [inline profiler/src/Demos/Samples.ExceptionGenerator/Properties/launchSettings.json:26]: maybe?
```suggestion
        "DD_PROFILING_EXCEPTION_ENABLED": "1",
        "DD_PROFILING_ALLOCATION_ENABLED": "1",
```

File: profiler/src/Demos/Samples.Website-AspNetCore01/Properties/launchSettings.json [hunk 5]
```diff
@@ -12,39 +12,53 @@
...
-        "COR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
+
+        "COMPlus_EnableDiagnostics": "1",
+
+
         "DD_PROFILING_ENABLED": "1",
-        "DD_DOTNET_PROFILER_HOME": "$(BuildOutputRoot)\\bin\\$(Configuration)-AnyCPU\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Managed\\",
-        "DD_INTERNAL_USE_DEVELOPMENT_CONFIGURATION": "true"
+        "DD_PROFILING_CPU_ENABLED": "1",
+        "DD_PROFILING_EXCEPTION_ENABLED": "1",
```
Comment (andrewlock) [inline profiler/src/Demos/Samples.Website-AspNetCore01/Properties/launchSettings.json:30]:
```suggestion
        "DD_PROFILING_EXCEPTION_ENABLED": "1",
        "DD_PROFILING_ALLOCATION_ENABLED": "1",
```

File: profiler/src/Demos/Samples.Website-AspNetCore01/Properties/launchSettings.json [hunk 6]
```diff
@@ -12,39 +12,53 @@
...
```
Comment (andrewlock) [inline profiler/src/Demos/Samples.Website-AspNetCore01/Properties/launchSettings.json:57]:
```suggestion
        "DD_PROFILING_EXCEPTION_ENABLED": "1",
        "DD_PROFILING_ALLOCATION_ENABLED": "1",
```

File: profiler/src/ProfilerEngine/DeployResources/DDProf-SetEnv.bat [hunk 7]
```diff
@@ -33,26 +33,26 @@ IF "%2"=="" (
     ECHO     Use the second command line parameter to specify the DD-Prof-DotNet Alpha Deploy Directory.
     ECHO     Using default.
 
-    SET DD_DOTNET_PROFILER_HOME=%~dp0
+    SET DD_DEPLOY_DIRECTORY=%~dp0
```
Comment (andrewlock) [inline profiler/src/ProfilerEngine/DeployResources/DDProf-SetEnv.bat:36]: Orrrr..... we could delete these files 😉

**Review by andrewlock — APPROVED** [review]
Thanks for clearing up my mess!

File: profiler/README.MD (hunk 1 above)
Comment (chrisnas) [reply profiler/README.MD:53]: This is the VS driven way to compile and debug.
Feel free to add the corresponding NUKE way to build!

File: profiler/src/Demos/NoOpWcfService/Properties/launchSettings.json (hunk 2 above)
Comment (chrisnas) [reply profiler/src/Demos/NoOpWcfService/Properties/launchSettings.json:13]: I did not update these WCF samples used to reproduce a customer issue a long time ago

File: profiler/src/Demos/Samples.Computer01/Properties/launchSettings.json (hunk 3 above)
Comment (chrisnas) [reply profiler/src/Demos/Samples.Computer01/Properties/launchSettings.json:26]: Yes it is  :^)
Allocation profiling is not yet available and I only updated demo apps that allocate memory (BuggyBits and Computer01)

File: profiler/src/Demos/Samples.ExceptionGenerator/Properties/launchSettings.json (hunk 4 above)
Comment (chrisnas) [reply profiler/src/Demos/Samples.ExceptionGenerator/Properties/launchSettings.json:26]: I added only the interesting profilers (no allocations here)

File: profiler/src/Demos/Samples.Website-AspNetCore01/Properties/launchSettings.json (hunk 6 above)
Comment (chrisnas) [reply profiler/src/Demos/Samples.Website-AspNetCore01/Properties/launchSettings.json:57]: Same

File: profiler/src/ProfilerEngine/DeployResources/DDProf-SetEnv.bat (hunk 7 above)
Comment (chrisnas) [reply profiler/src/ProfilerEngine/DeployResources/DDProf-SetEnv.bat:36]: I prefer to wait for Gregory to come back and decide what to do  ;^)

**Comment by chrisnas** [comment]
> Thanks for clearing up my mess!

My pleasure!
We should have done that a long time ago when the managed side of the profiler has been removed

**Review by gleocadie — APPROVED** [review]

**Pull request merged**